spec/users_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe 'GET #index' do
+    it 'returns http index success' do
+      get :index
+      expect(response).to have_http_status(:success)
+      expect(response.status).to eq(200)
+    end
```

Comment:
- [ ] Ass required for the project, kindly make sure each action has a is rendered. `index` and `show` actions for `User` and `Post` controllers are tested. Right now, I can see that every where, you only rendered the `:index` i.e. you should also have tests which render  the `:show` template. 
e.g.:
```ruby
...
describe 'GET /show' do
  ...
  expect(response).to render_template(:show)
  ...
end
```

Reply:
ok i change it